src/components/StarterKits/StarterKitsTable.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Kits from './_starter-kits.json';
+import Link from '../Link/Link';
+
+const StarterKitsTable = props => (
+    <div className="table-body">
+    { Kits.map((kit, i) => (
```

Comment:
We need prettier :D

Reply:
Sending a PR ASAP.

Reply:
This is our code copied over, probably vscode re-aligned based on his brains :D